images/macos/helpers/Tests.Helpers.psm1

```diff
@@ -1,3 +1,5 @@
+Import-Module "$PSScriptRoot/Common.Helpers.psm1"
```

Comment:
I think you should also update YAML template and remove step for publishing test results: https://github.com/actions/virtual-environments/blob/main/images.CI/macos/azure-pipelines/image-generation.yml#L124.

Currently, your builds have warning due to it: 
![image](https://user-images.githubusercontent.com/16715858/102450136-d3c93780-4046-11eb-9378-3ef53cd311d4.png)